util/design/data/prim_mubi_pkg.core.tpl

```diff
@@ -0,0 +1,45 @@
+CAPI=2:
```

Comment:
This is really a question from my interest rather than a review comment, but can splitting out `prim_mubi_pkg` come as an initial commit? (Or does it depend on the change in `pwrmgr` as well?)

Reply:
Done!